### PR TITLE
re-released all plugins with proper dependency on hydra-core

### PR DESCRIPTION
**Pull request by omry**

### CODE REVIEW

File: plugins/hydra_submitit_launcher/setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     LONG_DESC = fh.read()
     setup(
         name="hydra-submitit-launcher",
-        version="1.0.0rc1",
+        version="1.0.0rc333
```
Comment (jieru-hu) [inline plugins/hydra_submitit_launcher/setup.py:8]: why 333?

File: plugins/hydra_nevergrad_sweeper/setup.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
     LONG_DESC = fh.read()
     setup(
         name="hydra-nevergrad-sweeper",
-        version="1.0.0rc2",
+        version="1.0.0rc4",
```
Comment (jieru-hu) [inline plugins/hydra_nevergrad_sweeper/setup.py:9]: why the rc version increments by 2?

File: plugins/hydra_submitit_launcher/setup.py (hunk 1 above)
Comment (omry) [reply plugins/hydra_submitit_launcher/setup.py:8]: well shit, good catch.

File: plugins/hydra_nevergrad_sweeper/setup.py (hunk 2 above)
Comment (omry) [reply plugins/hydra_nevergrad_sweeper/setup.py:9]: I am keeping the repo version on the next version, not on the released version.

**Pull request merged**